### PR TITLE
NCAS-2374 #comment merged changes for sprint 92 that have already bee…

### PR DESCRIPTION
**Pull request by burtonquantil**
…n reviewed by our QA team

### CODE REVIEW

File: apispec/openapi.json [hunk 1]
```diff
@@ -22331,7 +23343,7 @@
...
+      "description": "A property is the configuration of one or more hostnames (domains) that need to be <a href=\"#tag/Deployment-Management\">deployed</a> to the CDN360 servers. A property can have multiple versions. A property version that has been deployed cannot be updated or <a href=\"#tag/Property-Validation\">validated</a> again. You can <a href=\"#operation/createPropertyVersion\">create a new version of a property</a> by making a clone (or copy) of an existing version. <p>The following figure shows the steps for creating an acceleration service for a hostname.</p><img src=\"https://docs.cdnetworks.com/static/content-assets/mileweb/docs-content-cdn/docs/resources/images/getting-started-flowchart.png\">"
```
Comment (nibinqtl) [inline apispec/openapi.json:23346]: For the image src, can we remove "https://docs.cdnetworks.com" and start from the URI?

File: apispec/openapi.json [hunk 2]
```diff
@@ -176,7 +193,7 @@
                         "edgeLogic": {
                           "type": "string",
                           "maxLength": 200000,
-                          "description": "Refer to https://docs.quantil.com/edgelogic"
+                          "description": "Refer to https://docs.cdnetworks.com/cdn/docs/edge-logic/intro"
```
Comment (nibinqtl) [inline apispec/openapi.json:196]: Can we remove "https://docs.cdnetworks.com" and start from the URI?

File: apispec/openapi.json [hunk 3]
```diff
@@ -435,6 +452,11 @@
                               "action"
                             ]
                           }
+                        },
+                        "disableCertAutomation": {
+                          "type": "boolean",
+                          "description": "If you plan to allow CDNetworks to renew your TLS certificate through Let's Encrypt, set this to <i>false</i>. \n If you plan to handle certificate renewal yourself,  set this to <i>true</i>. For more information about our support for certificate auto-renewal, refer to the description of the <i>autoRenew</i> field in the <a href=\"#operation/createCertificate\">Create a certificate API</a>.",
```
Comment (nibinqtl) [inline apispec/openapi.json:458]: Please update the description based on https://docs.google.com/document/d/19e8APetleIp-8hc3Hy1qYhrThsWqN803VZYswzjX1lg/edit

**Review by nibinqtl — CHANGES_REQUESTED** [review]

File: apispec/openapi.json (hunk 1 above)
Comment (burtonquantil) [reply apispec/openapi.json:23346]: Yes, I believe this will work because it is part of an img tag.  I will fix this.

**Comment by burtonquantil** [comment]
I made your requested changes in a new pull request because I had trouble figuring out how to resolve some GitHub conflict.

**Pull request closed without merging**